apps/condo/domains/ticket/schema/UserTicketCommentRead.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Generated by `createschema ticket.UserTicketCommentRead 'user:Relationship:User:CASCADE; ticket:Relationship:Ticket:CASCADE; readResidentCommentAt:DateTimeUtc;'`
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File } = require('@keystonejs/fields')
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/UserTicketCommentRead')
+
+
+const UserTicketCommentRead = new GQLListSchema('UserTicketCommentRead', {
+    schemaDoc: 'Last date the comment was read in the ticket by the user',
```

Comment:
Current documentation means specific comment, which does not matches current implemented schema. It looks, like this record just tracks reading action of any ticket comment, not a specific one. Could you please reflect it in documentation?